src/main/java/org/broadinstitute/consent/http/db/MailMessageDAO.java

```diff
@@ -32,4 +35,22 @@ Integer insert(@Nullable @Bind("entityReferenceId") String entityReferenceId,
                 @Nullable @Bind("sendGridResponse") String sendGridResponse,
                 @Nullable @Bind("sendGridStatus") Integer sendGridStatus,
                 @Bind("createDate") Instant createDate);
+
+    @SqlQuery("""
+             SELECT entity_reference_id, email_entity_id, vote_id, user_id, email_type, date_sent, email_text, sendgrid_response, sendgrid_status, create_date FROM email_entity e
+             WHERE email_type = :emailType
+             ORDER BY create_date DESC
+             OFFSET :offset
+             LIMIT :limit
+             """)
+    List<MailMessage> fetchMessagesByType(@Bind("emailType") Integer emailType, @Bind("limit") Integer limit, @Bind("offset") Integer offset);
+
+    @SqlQuery("""
+             SELECT entity_reference_id, email_entity_id, vote_id, user_id, email_type, date_sent, email_text, sendgrid_response, sendgrid_status, create_date FROM email_entity e
+             WHERE create_date BETWEEN SYMMETRIC :start AND :end 
```

Comment:
BETWEEN SYMMETRIC allows for the end date to be before the start date.  The values are sorted before the expression is executed.  See: [https://www.postgresql.org/docs/current/functions-comparison.html#:~:text=datatype-,BETWEEN%20SYMMETRIC,-datatype%20AND%20datatype](https://www.postgresql.org/docs/current/functions-comparison.html#:~:text=datatype-,BETWEEN%20SYMMETRIC,-datatype%20AND%20datatype)